jobbergate-api/jobbergate_api/apps/job_script_templates/schemas.py

```diff
@@ -154,15 +157,24 @@ class Config:
         schema_extra = job_template_meta_mapper
 
 
-class JobTemplateResponse(TableResource):
-    """Schema for the request to create a job template."""
+class JobTemplateListView(TableResource):
+    """Schema for the response to get a list of entries."""
 
     identifier: Optional[str]
-    template_vars: Optional[dict[str, Any]] = {}
-
-    template_files: list[TemplateFileResponse] = []
-    workflow_files: list[WorkflowFileResponse] = []
 
     class Config:
-        orm_mode = True
         schema_extra = job_template_meta_mapper
+
+
+class JobTemplateDetailedView(JobTemplateListView):
+    """
+    Schema for the request to an entry.
+
+    Notice the files default to None, as they are not always requested, to differentiate between
+    an empty list when they are requested, but no file is found.
+    """
+
+    template_vars: Optional[dict[str, Any]] = {}
```

Comment:
typing on this one should also be changed to: `dict[str, Any] | None`

Reply:
good catch, solved in [ba3b893](https://github.com/omnivector-solutions/jobbergate/pull/292/commits/ba3b89379ecb2c8ee71223f18d7b86a89bc5fca9)